src/js/controller.js

```diff
@@ -19,12 +20,24 @@ const controlAddNote = function() {
 };
 
 const controlDeleteNote = function(id) {
+  DeleteConfirmationView.toogleWindow();
+  model.state.noteToDelete = id;
+};
+
+const controlDeleteConfirmation = function() {
+  DeleteConfirmationView.toogleWindow();
+  let id = model.state.noteToDelete;
   if (id === model.state.pinNoteID) model.state.pinNoteID = null;
   model.removeNoteFromFolder(id);
   model.deleteNote(id);
+  gitmodel.state.noteToDelete = null;
   model.mapSortFunc.get(model.state.currentSorting)();
   NotesView.render(model.state.currentNotesView);
-};
+}
+
+const controlDeleteCancel = function() {
+  DeleteConfirmationView.toogleWindow();
```

Comment:
When delete operation is rejected you should reset a model.state.noteToDelete variable